sparkly/session.py

```diff
@@ -72,15 +74,33 @@ class MySession(sparkly.SparklySession):
             Key - a name of the function,
             Value - either a class name imported from a JAR file
                 or a tuple with python function and its return type.
+        app_id_tempate (str|None): if set and nonempty, generate the `spark.app.id` with
```

Comment:
Should we mention that if `spark.app.id` is provided explicitly, it overrides whatever the template generates?